stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -56,19 +73,44 @@ public ResponseBodyData readFile(String requestId) throws IOException {
       }
       ResponseBodyData bodyData = new ResponseBodyData();
       bodyData.base64Encoded = firstByte != 0;
-      bodyData.data = readContentsAsUTF8(in);
+
+      if (mRequestIdMap.containsKey(requestId)) {
+        AsyncPrettyPrinter asyncPrettyPrinter = mRequestIdMap.get(requestId);
+        bodyData.data = prettyPrintContentWithTimeOut(asyncPrettyPrinter, in);
+      } else {
+        bodyData.data = readContentsAsUTF8(in);
+      }
       return bodyData;
     } finally {
       in.close();
     }
   }
 
-  private static String readContentsAsUTF8(InputStream in) throws IOException {
+  public static String readContentsAsUTF8(InputStream in) throws IOException {
```

Comment:
This should be moved to `stetho.common` now that it is being used by more than one distinct caller.  Don't just mark private functions public because it is the least amount of typing :)